src/main/java/be/xplore/price_scraper/domain/shopping/ItemPrice.java

```diff
@@ -0,0 +1,19 @@
+package be.xplore.price_scraper.domain.shopping;
+
+import jakarta.persistence.*;
+import lombok.RequiredArgsConstructor;
+
+import java.sql.Timestamp;
+
+@RequiredArgsConstructor
+@Entity
+public class ItemPrice {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
```

Comment:
always question if an auto generated number is needed
in this case it might be also possible to use item name + shop name or something like that

Reply:
Is this done in order to reduce data usage?